rero_ils/modules/acq_orders/serializers.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Acquisition order serialization."""
+
+from invenio_records_rest.serializers.response import search_responsify
+
+from ..libraries.api import Library
+from ..serializers import JSONSerializer, RecordSchemaJSONV1
+
+
+class AcqOrderJSONSerializer(JSONSerializer):
+    """Mixin serializing records as JSON."""
+
+    def post_process_serialize_search(self, results, pid_fetcher):
+        """Post process the search results."""
+        # Add library name
+        for lib_term in results.get('aggregations', {}).get(
+                'library', {}).get('buckets', []):
+            pid = lib_term.get('key')
+            name = Library.get_record_by_pid(pid).get('name')
```

Comment:
if the purpose of this is just for display in the UI please do it in the UI using the getRecord pipe.

Reply:
It's used to build library facet for UI. I don't know how to use getRecord pipe on facet.

Reply:
Ok it makes sense.